### DISCUSSION TITLE
Missing Jaeger icons in Linkerd UI

### DISCUSSION THREAD

**Discussion by hongbo-miao** [linkerd/linkerd2#6464] [Q&A] [answered]
I saw in the version 2.10 doc
https://linkerd.io/2.10/tasks/distributed-tracing/

there should have Jaeger icons.

I can see all traces in Jaeger UI, but I do not see those Jaeger icons in Linkerd UI.

![image](https://user-images.githubusercontent.com/3375461/125209115-3aef5780-e2c9-11eb-97d6-5d855b5c05ef.png)

Any idea? Thanks

BTW, I saw in version 2.9 https://linkerd.io/2.9/tasks/distributed-tracing/

I need a config.yaml
```
tracing:
  enabled: true
```
Just want to confirm, I do not need it in 2.10, right?

**Comment by alpeb** [comment, marked as answer]
Thanks for the feedback @Hongbo-Miao , that's a good find :+1: 
Indeed, that 2.9 config is not relevant for 2.10. For the 2.10 dashboard to show the jaeger links it requires that the `web` Deployment in the `linkerd-viz` namespace contains a flag pointing to the jaeger UI: `jaeger-addr=jaeger.linkerd-jaeger:16686`. That can be edited manually, or through the CLI (or equivalently through Helm) by upgrading the jaeger extension with `linkerd jaeger install --set jaegerUrl=xxx`. I'll open a ticket to clarify that in the docs.
OTOH, as a further improvement, this could be handled automatically by the dashboard component by dynamically figuring if the jaeger extension is installed...

Edit: Issue created at #6470

**Reply by hongbo-miao** [reply]
Thanks @alpeb ! Just confirm, do you mean

```shell
linkerd viz install --set jaegerUrl=jaeger.linkerd-jaeger:16686
```

or

```shell
linkerd jaeger install --set jaegerUrl=jaeger.linkerd-jaeger:16686
```

?

Because we want to let Linkerd UI to point to Jaeger.

**Reply by hongbo-miao** [reply]
Oh, I guess it is just a typo.

Just confirmed the correct one is `linkerd viz install --set jaegerUrl=jaeger.linkerd-jaeger:16686`. Now I can see the Jaeger icons in Linkerd UI:

![image](https://user-images.githubusercontent.com/3375461/125315847-1a2b0e80-e36a-11eb-8629-4375997f212c.png)
